frontend/components/inputs/RangeSliderInput.js

```diff
@@ -0,0 +1,49 @@
+import React, {useState} from "react";
+import PropTypes from "prop-types";
+import Slider from '@mui/material/Slider';
+
+const RangeSliderInput = (
+    {units, minValue, maxValue, updateValues, step, enabled=true, values=[minValue, maxValue]}) => {
+    const [sliderValues, setSliderValues] = useState(values);
+
+
+    const updateSlider = (event, newValues) => {
+        event.preventDefault();
+        setSliderValues(newValues);
+        updateValues(newValues);
+    };
+
+    return(enabled
```

Comment:
`enabled` is a prop and we know the value at the beginning of the function, which means we don't need to wait until this return statement to check its value. Instead, I'd stick the following at the very top of the function:

```
if (!enabled) return null;
```

Couple of benefits: our return statement becomes a little clearer (since we remove the conditional) and -- more importantly -- we shortcut out of the function early and avoid unnecessarily doing any setup work.